test/Microsoft.AspNet.SignalR.ServiceBus.Tests/ServiceBusConnectionFacts.cs

```diff
@@ -37,7 +37,7 @@ public void RetryRetriesActionOnStandardException()
             {
                 ++called;
                 if (called < 2)
-                    throw new Exception("Error");
```

Comment:
This is wrong. This `Exception` ctor takes a message as the parameter and not parameter name. This change does not add any value and actually makes the code more confusing. Same applies to changes below. 

(From what I see a good change in this file would be to remove `ref` from the `ref int called` parameter on the `RetryHelper` method - it does not seem to be changed inside the method and therefore does not have to be `ref` -  the tests still must build and pass after the change).